ui/lib/utils/table-columns.tsx

```diff
@@ -33,21 +33,27 @@ for (const key in translations) {
   })
 }
 
-export function useDummyColumn(): IColumn {
+const DEF_TRANSKEY_PREFIX = 'component.commonColumn.'
+
+function TransText({ transKey }: { transKey: string }) {
+  const { t } = useTranslation()
+  return <span>{t(transKey, '')}</span>
+}
+
+export function dummyColumn(): IColumn {
   return {
     name: '',
     key: 'dummy',
     minWidth: 28,
     maxWidth: 28,
     columnActionsMode: ColumnActionsMode.disabled,
-    onRender: (rec) => null,
+    onRender: (_rec) => null,
   }
 }
 
-export function useFieldsKeyColumn(translationPrefix: string): IColumn {
-  const { t } = useTranslation()
+function fieldsKeyColumn(transKeyPrefix: string): IColumn {
   return {
-    name: t('component.commonColumn.name'),
+    name: (<TransText transKey={`${DEF_TRANSKEY_PREFIX}name`} />) as any,
```

Comment:
In this way, it will keep the same style for all columns util:

```js
function commonColumnName(fieldName: string): any {
  return <TextWithInfo.TransKey transKey={`component.commonColumn.${fieldName}`} />
}

...

return {
  name: commonColumnName('name'),
  ...
}
```

Reply:
Yep, that's would be better, we need a more function, will update it.